api-reference/media-api/README.md

```diff
@@ -120,9 +120,13 @@ All media APIs accept JSON-encoded request bodies and return JSON-encoded respon
 | thumbnail         | In the case of an image, a small thumbnail URL.                                                                                                                                                                                         |
```

Comment:
Didn't update the sample response on https://docs.imagekit.io/api-reference/media-api/get-file-details & other pages under the Media API category since the responses are for an image file. Adding keys like `audio_codec` would make the sample response confusing.

So I'm only update the structure which lists out all keys for Media & Metadata API.